straxen/plugins/peaklet_processing.py

```diff
@@ -702,7 +721,10 @@ def infer_dtype(self):
         return strax.unpack_dtype(self.deps['peaklets_he'].dtype_for('peaklets_he'))
 
     def compute(self, peaklets_he):
-        return super().compute(peaklets_he)
+        # There are not any lone hits for the high energy channel, 
+        #  so create a dummy for the compute method.
+        lone_hits = np.zeros(0, dtype=strax.hit_dtype)
```

Comment:
we could easily make lone_hits_he if needed, but since no one is looking at it anyway. I guess this is fine

Reply:
I am not sure if it would be worth adding. I would not really expect to see some for the high energy channels. Though I never tested.

Reply:
Yeah, I don't think it's worth it

If the threshold for seeing something something is higher, the chances are that at least similar amount of single PMTs are seeing something right for the HE channels (also because it's only half of the TPC)?